### PR TITLE
New package: keepassx2-alpha6

### PR DESCRIPTION
**Pull request by Grauwolf**

### CODE REVIEW

File: srcpkgs/keepassx2/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
+# Template file for 'keepassx2'
+_realname="keepassx"
+_realversion="2.0-alpha6"
+pkgname="${_realname}2"
+version="2.0"
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config"
+makedepends="cmake qt-devel libgcrypt-devel zlib-devel"
```
Comment (xtraeme) [inline srcpkgs/keepassx2/template]: cmake must be in hostmakedepends instead.

**Comment by logenkain** [comment]
On Wed, Apr 01, 2015 at 12:33:50PM -0700, Farhad Shahbazi wrote:
>   * New package: keepassx2-alpha6
Glad someone is finally packaging this.  No idea why I didn't do it
already.

**Comment by xtraeme** [comment]
Looks ok to be your first template, thanks :+1:

**Comment by xtraeme** [comment]
Applied, thanks.

**Pull request closed without merging**